source/includes/assessment/_test_status.md

```diff
@@ -51,7 +51,7 @@ The response to a `test_status` request should contain a JSON object in its body
 
 Property Name | Type | Required | Description
 -------------- | -------------- | -------------- | --------------
-partner_status | String | Yes | Describes the current state of the test instance. If the test has been completed and results are available, this value should be **"complete."** We will continue to poll until the status is "complete."
+partner_status | String | Yes | Describes the current state of the test instance. If the test has been completed and results are available, this value should be **"complete."** We will continue to poll until the status is "complete," or until 8 weeks has passed since the test was sent.
```

Comment:
I would take the punctuation out of the status quotes. I know it's grammatically correct but usually in docs it's better to tell engineers _exactly_ what would be in the quotes and let grammar take a backseat.